freyja/read_analysis_tools.py

```diff
@@ -0,0 +1,268 @@
+import os
+import re
+import pysam
+
+
+def extract(query_mutations, input_bam, output):
+    # Load data
+    with open(query_mutations) as infile:
+        mutations = infile.read().splitlines()
+
```

Comment:
probably worth making a comment here that we're taking care of the case of less than 3 lines in the query input file. Also, what happens if the user provides a single line, but that line is deletions or insertions?